### DISCUSSION TITLE
Spatial Transcriptomics

### DISCUSSION THREAD

**Discussion by RamRS** [alexdobin/STAR#1521] [Q&A]
Is it possible to use STAR instead of space ranger for spatial transcriptomics data? It looks like SR, much like Cell Ranger, uses STAR under the hood, so being able to use STAR directly would help us customize analyses.

**Comment by alexdobin** [comment]
Hi Ram,

it should be possible, since the barcoding scheme must be very similar, but I have not done it myself.
If you decide to try it, will be happy to help.
